test/OpenTelemetry.Extensions.AWS.Tests/AWSXRayIdGeneratorTests.cs

```diff
@@ -23,12 +23,7 @@ public void TestGenerateTraceIdForRootNode()
             activity.Start();
 
             Assert.NotEqual(originalTraceId, activity.TraceId);
-#if NET6_0_OR_GREATER
-            // the net6.0 version of AWSXRayIdGenerator uses Activity.TraceIdGenerator, which does not change the parent ID
             Assert.Equal(originalParentSpanId, activity.ParentSpanId);
-#else
```

Comment:
Was this silently failing up until now?

Reply:
I think so? I tried to look back at the run where it was introduced to see how it got in initially but detailed logging isn't maintained that far back.

The actual code always sets [parentSpanId to default](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/fb693359f0d91619b904f55bf1a3eed16958610a/src/OpenTelemetry.Extensions.AWS/AWSXRayIdGenerator.cs#L41) so I'm not sure how it ever ended up something other than what the root has.